roles/foreman_installer/defaults/main.yml

```diff
@@ -19,6 +19,9 @@ foreman_installer_module_prs_strategy: "merge"
 # Comma-separated list of "organization/module/branch", e.g. "theforeman/foreman/foo,theforeman/katello/bar"
 foreman_installer_module_branches: []
 
+foreman_installer_from_source: False
+foreman_installer_git_pr_number:
```

Comment:
Can you use `foreman_installer_git_pr_number` without also using it from source?

Reply:
No. The relevant task only gets included when building the installer from source. I'm planning to add comments which document what these variables do and how to use them.

Reply:
A nice improvement would be that if I set the value of `foreman_installer_git_pr_number` it automatically sets `foreman_installer_from_source` to true.